.claude/hooks/utils/llm/ollama_provider.py

```
@@ -0,0 +1,193 @@
+            #     pass
+            break
+
+    api_key = os.getenv("OLLAMA_API_KEY")
+    base_url = os.getenv("OLLAMA_BASE_URL", "https://ollama.com")
+    model = os.getenv("OLLAMA_MODEL", "gpt-oss:120b")
+    
+    # Log only if API key is missing for debugging
+    if not api_key:
+        try:
+            with open(".claude/logs/summarizer.log", "a") as f:
+                f.write(f"[DEBUG] Ollama API key not found in .env\n")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)